distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -402,9 +408,10 @@ type Column
 
              example_gt = Examples.integer_column > 1
     > : Column | Any -> Column
-    > self other = Value_Type.expect_comparable self other <|
-        Incomparable_Values.handle_errors <|
-            run_vectorized_binary_op self Java_Storage.Maps.GT fallback_fn=(>) other expected_result_type=Value_Type.Boolean
+    > self other = Value_Type.expect_comparable self other <| Incomparable_Values.handle_errors <|
+        new_name = naming_helper.binary_operation_name ">" self other
+        call_comparator self other new_name Comparators.GT <|
+            run_vectorized_binary_op self Java_Storage.Maps.GT new_name=new_name fallback_fn=(>) other expected_result_type=Value_Type.Boolean
```

Comment:
Having `call_comparator` AND `run_vectorized_binary_op` seems very redundant, I assume that in later iterations we plan to remove the latter?

Reply:
Yes it allows me to turn on Value_Type by Value_Type